src/backend/app/app.py

```diff
@@ -57,13 +112,13 @@ def login():
         email = request.form['email']
         password = request.form['password']
 
-        # Retrieve the user from the database
+       
         with get_db() as conn:
             user = conn.execute('SELECT * FROM users WHERE email = ?', (email,)).fetchone()
 
         if user and user['password'] == password:
             flash(f"Welcome back, {user['username']}!", 'success')
-            return redirect(url_for('queryforge'))  # Redirect to the front page
+            return redirect(url_for('queryforge'))  
         else:
             flash('Invalid credentials! Please try again.', 'danger')
```

Comment:
Found the problem, index and login routes got merged here. Fixed it

Reply:
Okay.